### DISCUSSION TITLE
Left Half/Right Half don't work for me - feels like applications intercept before the Rectangle Pro

### DISCUSSION THREAD

**Discussion by toomasr** [rxhanson/RectanglePro-Community#145] [Q&A] [answered]
I just purchased the pro version and uninstall the regular one. I'm trying to use the "Left Half" shortcut but it does nothing with the window but actually cycles through tabs in Chrome and Firefox.

Remove Rectangle.app. Purchase Rectangle Pro. Set it up and add a shortcut option-command-arrow-left for "Left Half". Go to any app with a GUI and press the combo and something else happens.

It almost feels that the application gets to handle the shortcut before Rectangle Pro steps in.

Am I doing something wrong? Missing a configuration option? How to get the desired functionality?

**Comment by rxhanson** [comment, marked as answer]
Sounds like you're looking for Spectacle default shortcuts, for cmd+option+arrow keys. To get this in Rectangle Pro, execute the following terminal command and restart the app:

`defaults write com.knollsoft.Hookshot alternateDefaultShortcuts -bool true`

The defaults in Pro are the "recommended" ones in Rectangle. I think most users that use the Spectacle shortcuts in Rectangle choose to import Rectangle shortcuts on first launch of Pro, which should carry over the Spectacle ones.

**Comment by toomasr** [comment]
Thank you, everything is solved now!
